cloudevents/event_test.go

```diff
@@ -255,13 +277,13 @@ func TestExtensionExtraction(t *testing.T) {
 		t.Fatal("Failed to parse request", err)
 	}
 
-	expectedExtensions := map[string]interface{}{
+	expectedV01Extensions := map[string]interface{}{
 		"Prop1": "value1",
 		"Prop2": map[string]interface{}{
 			"nestedProp": "nestedValue",
 		},
 	}
-	if !reflect.DeepEqual(expectedExtensions, ctx.Extensions) {
-		t.Fatalf("Did not parse expected extensions. Wanted=%v; got=%v", expectedExtensions, ctx.Extensions)
+	if !reflect.DeepEqual(expectedV01Extensions, ctx.AsV01().Extensions) {
```

Comment:
Suggest: `cmp.Equal`?

Reply:
Switched to `cmp.Diff` and compare with empty string.